Source/WebCore/rendering/RenderThemeIOS.mm

```diff
@@ -1358,6 +1364,36 @@ static bool shouldUseConvexGradient(const Color& backgroundColor)
     return systemFocusRingColor();
 }
 
+Color RenderThemeIOS::insertionPointColor()
+{
+    if (!cachedInsertionPointColor().has_value())
+        cachedInsertionPointColor() = Color::transparentBlack;
+    return *cachedInsertionPointColor();
+}
+
+Color RenderThemeIOS::autocorrectionReplacementMarkerColor(const RenderText& renderer, OptionSet<StyleColorOptions>) const
```

Comment:
Nit - would it be cleaner to just pass in a `renderer` here, and use `renderer.styleColorOptions()` to grab the color options from it? (So that the function never takes a set of `StyleColorOptions` that doesn't match the `renderer`).

Reply:
I was already using `renderer.styleColorOptions()` in the method body – removed the parameter.